assets/js/base/context/providers/store-notices/components/store-notices-container.js

```diff
@@ -2,16 +2,25 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { useDispatch, useSelect } from '@wordpress/data';
 import classnames from 'classnames';
 import { Notice } from 'wordpress-components';
-import { useDispatch, useSelect } from '@wordpress/data';
-
+import { sanitize } from 'dompurify';
 /**
  * Internal dependencies
  */
 import './style.scss';
 import { useStoreNoticesContext } from '../context';
 
+const ALLOWED_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
+const ALLOWED_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];
+
+const sanitizeHTML = ( html ) => {
+	return {
+		__html: sanitize( html, { ALLOWED_TAGS, ALLOWED_ATTR } ),
+	};
+};
```

Comment:
@hsingyuc While working on #7147, I was reusing this part. When looking at `ALLOWED_TAGS` and `ALLOWED_ATTR`, I noticed that they're not working as expected. https://github.com/cure53/DOMPurify#can-i-configure-dompurify shows the following example:

```js
// allow only <b> and <q> with style attributes
var clean = DOMPurify.sanitize(dirty, {ALLOWED_TAGS: ['b', 'q'], ALLOWED_ATTR: ['style']});
```

Looking at the implementation above, I'd say that the current implementation would lead to this function call:

```js
const sanitizeHTML = ( html ) => {
	return {
		__html: sanitize( html, { 
			[ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ],
			[ 'target', 'href', 'rel', 'name', 'download' ]
		} ),
	};
};
```

I believe the function should be called like this instead:

```js
const sanitizeHTML = ( html ) => {
	return {
		__html: sanitize( html, { 
			ALLOWED_TAGS: [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ],
			ALLOWED_ATTR: [ 'target', 'href', 'rel', 'name', 'download' ]
		} ),
	};
};
```

If that assumption is correct, I suggest the following change:

```suggestion
const tags = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
const attr = [ 'target', 'href', 'rel', 'name', 'download' ];

const sanitizeHTML = ( html ) => {
	return {
		__html: sanitize( html, { ALLOWED_TAGS: tags, ALLOWED_ATTR: attr } ),
	};
};
```

cc: @opr and @wavvves

Reply:
@nielslange, in ES6, it converts to below. If converted to the way you wrote, I believe we will get an error.
``` 
{ 
    ALLOWED_TAGS:   [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ]
} 
```